app/editor/src/components/form/wysiwyg/Wysiwyg.tsx

```diff
@@ -44,7 +52,7 @@ export const Wysiwyg: React.FC<IWysiwygProps> = ({ fieldName, label, required, e
 
   React.useEffect(() => {
     if (!!id) {
-      setState({ ...state, html: values[fieldName] as string });
+      setState({ ...state, html: values[fieldName!] as string });
```

Comment:
This can fail because `fieldName` can be undefined.

Reply:
This update is due to the refactoring for removing the interface as mentioned above, and fieldName was initially defined as a mandatory property which now has been updated as an optional property, so in this case it won't be undefined. Thanks

Reply:
code updated again, thanks